### DISCUSSION TITLE
How to debounce loading remote data?

### DISCUSSION THREAD

**Discussion by gubler** [orchidjs/tom-select#430] [Q&A] [answered]
I am successfully loading remote data, but I don't know how to set the delay before a request is sent.

Currently, unless I type very fast, I get a request per keystroke to my server. With other libraries I know there is a way to "debounce" the request that waits until a certain amount on time passes with no new key-strokes before sending a request.

I also don't know if there is a way to cancel previous, running requests if I pause, a request is sent, and then start typing before a result is returned.

**Comment by oyejorge** [comment, marked as answer]
https://tom-select.js.org/docs/#loadthrottle

**Reply by gubler** [reply]
Thank you very much - I didn't know the right term to look for in the docs.
